src/app/arena/page.tsx

```diff
@@ -0,0 +1,260 @@
+"use client";
```

Comment:
pages should be rendered server side, cretate components to shift "use client" logic in it